htmlbars-version/app/components/focusing-outlet.js

```diff
@@ -0,0 +1,14 @@
+import Ember from "ember";
+import FocusingOutletMixin from 'ember-a11y/mixins/focusing';
+
+const { Component } = Ember;
+
+let FocusingOutlet = Component.extend(FocusingOutletMixin, {
+  positionalParams: ['inputOutletName'] // needed for Ember 1.13.[0-5] and 2.0.0-beta.[1-3] support
```

Comment:
We should go out of our way to make sure that everybody gets accessibility. I'm willing to deal with that.